src/main/java/com/gradle/CustomBuildScanEnhancements.java

```diff
@@ -106,6 +110,10 @@ private void execute() {
                 tagIde("IntelliJ IDEA", props.get(SYSTEM_PROP_IDEA_VERSION).get());
             } else if (props.get(SYSTEM_PROP_ECLIPSE_BUILD_ID).isPresent()) {
                 tagIde("Eclipse", props.get(SYSTEM_PROP_ECLIPSE_BUILD_ID).get());
+            } else if (props.get(ENV_VAR_VSCODE_PID).isPresent()) {
+                tagIde("VS Code", "");
+            } else if (props.get(ENV_VAR_VSCODE_INJECTION).isPresent()) {
+                tagIde("VS Code", "");
```

Comment:
Can you `||` this like you did [here](https://github.com/gradle/common-custom-user-data-gradle-plugin/blob/a1904c45915b043de479ec386c475301f8f0f04f/src/main/java/com/gradle/CustomBuildScanEnhancements.java#L142-L144)? 

```suggestion
            } else if (props.get(ENV_VAR_VSCODE_PID).isPresent() || props.get(ENV_VAR_VSCODE_INJECTION).isPresent()) {
                tagIde("VS Code", "");
```